tests/bsim/bluetooth/host/att/pipeline/dut/src/main.c

```diff
@@ -151,16 +151,10 @@ static struct bt_conn *connect(void)
 {
 	int err;
 	struct bt_conn *conn;
-	struct bt_le_scan_param scan_param = {
-		.type = BT_LE_SCAN_TYPE_ACTIVE,
-		.options = BT_LE_SCAN_OPT_NONE,
-		.interval = BT_GAP_SCAN_FAST_INTERVAL,
-		.window = BT_GAP_SCAN_FAST_WINDOW,
-	};
```

Comment:
Since you've updated some BSIM tests, consider if the mesh implementation and test should also use these new macros as Mesh also uses continuous scan:
```c
#define BT_MESH_SCAN_INTERVAL_MS 30
#define BT_MESH_SCAN_WINDOW_MS   30
```

Reply:
Then we will have more code for unclear reasons. I'm fine with what we have in mesh right now. Mesh scanning parameters are not something that customers (or anybody) should ever change.

Reply:
Yep, we keep short scanner window on purpose. If there was a packet collision, then scanner has an opportunity to receive the same packet on another channel if the window is short. With long windows, the mesh devices become to be quite deaf in the network with intensive traffic. 
Kind of agree with Pavel. Leave mesh to keep its own scanner settings.